### DISCUSSION TITLE
Fprime 3.1 release timeframe

### DISCUSSION THREAD

**Discussion by ebenezerarunkumar** [nasa/fprime#1457] [Q&A] [answered]
Hi! 

The Fprime team had mentioned before that an update(3.1) to fprime was in the works. Out of curiosity, is there an expected timeframe on this that we can expect?

Our team at Georgia Tech is currently upgrading to fprime 3.0, and we were just wondering if we should be on the lookout for 3.1 soon?

Thanks,
Ebenezer

**Comment by LeStarch** [comment, marked as answer]
We are getting pretty close.  The work is tracked here: https://github.com/nasa/fprime/projects/3, starting in the second column "Unassigned".  CCB are tickets not approved for this release yet.
